tests/unit_tests/services/test_dds.py

```diff
@@ -210,38 +310,75 @@ def test_create_project_token_file(self):
-                new_callable=AsyncMock) as mock_run,\
-                patch(
-                    'delivery.services.dds_service'
-                    '.DDSService._parse_dds_project_id'
+                    'delivery.models.project'
+                    '.DDSProject._parse_dds_project_id'
                     ) as mock_parse_dds_project_id:
-            mock_run.return_value.status_code = 0
             mock_parse_dds_project_id.return_value = "snpseq00001"
 
-            yield self.dds_service.create_dds_project(
+            yield DDSProject.new(
                     project_name,
                     project_metadata,
-                    token_path)
+                    auth_token=self.token_file.name,
+                    dds_service=self.dds_service)
 
-            mock_run.assert_called_once_with([
+            self.mock_dds_runner.run.assert_called_with([
                 'dds',
-                '--token-path', token_path,
+                '--token-path', self.token_file.name,
                 '--log-file', '/foo/bar/log',
                 '--no-prompt',
                 'project', 'create',
-                '--title', project_name,
+                '--title', project_name.replace('-', ''),
                 '--description', f'"{project_metadata["description"]}"',
                 '-pi', project_metadata['pi'],
                 '--owner', project_metadata['owners'][0],
                 '--researcher', project_metadata['researchers'][0],
                 '--researcher', project_metadata['researchers'][1],
                 ])
-            self.mock_dds_project_repo.add_dds_project\
-                .assert_called_once_with(
-                    project_name=project_name,
-                    dds_project_id=mock_parse_dds_project_id.return_value)
+
+    @gen_test
+    def test_release_project(self):
+        project_id = 'snpseq00001'
+        deadline = '90'
+        dds_project = DDSProject(
+                dds_service=self.dds_service,
+                auth_token=self.token_file.name,
+                dds_project_id=project_id)
+
+        yield dds_project.release(deadline=deadline)
+
+        self.mock_dds_runner.run.assert_called_with([
+            'dds',
+            '--token-path', self.token_file.name,
+            '--log-file', '/foo/bar/log',
+            '--no-prompt',
+            'project', 'status', 'release',
+            '--project', project_id,
+            '--no-mail',
+            '--deadline', deadline,
+            ])
+
+    @gen_test
+    def test_get_dds_project_title(self):
+        mock_dds_project = [{
+                "Access": True,
+                "Last updated": "Fri, 01 Jul 2022 14:31:13 CEST",
+                "PI": "matilda.aslin@medsci.uu.se",
+                "Project ID": "snpseq00025",
+                "Size": 25856185058,
+                "Status": "In Progress",
+                "Title": "AB1234"
+                }]
+
+        with patch( 'delivery.models.project.DDSProject._run',
```

Comment:
```suggestion
        with patch('delivery.models.project.DDSProject._run',
```
Should the space be there?

Reply:
Yes, good catch!